service-recovery.ipynb

```diff
@@ -0,0 +1,672 @@
+       "      <td>501</td>\n",
+       "      <td>Weekday</td>\n",
+       "      <td>42</td>\n",
+       "      <td>101</td>\n",
+       "      <td>58.415842</td>\n",
+       "      <td>58.415842</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>313</th>\n",
+       "      <td>96</td>\n",
+       "      <td>Weekday</td>\n",
+       "      <td>27</td>\n",
+       "      <td>57</td>\n",
+       "      <td>52.631579</td>\n",
+       "      <td>52.631579</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>229</th>\n",
+       "      <td>601</td>\n",
+       "      <td>Weekday</td>\n",
+       "      <td>114</td>\n",
+       "      <td>228</td>\n",
+       "      <td>50.000000</td>\n",
+       "      <td>50.000000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>266</th>\n",
+       "      <td>71</td>\n",
+       "      <td>Weekday</td>\n",
+       "      <td>31</td>\n",
+       "      <td>60</td>\n",
+       "      <td>48.333333</td>\n",
+       "      <td>48.333333</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>238</th>\n",
+       "      <td>605</td>\n",
+       "      <td>Weekday</td>\n",
+       "      <td>48</td>\n",
+       "      <td>91</td>\n",
+       "      <td>47.252747</td>\n",
+       "      <td>47.252747</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>...</th>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "      <td>...</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>227</th>\n",
+       "      <td>60</td>\n",
+       "      <td>Weekday</td>\n",
+       "      <td>240</td>\n",
+       "      <td>187</td>\n",
+       "      <td>-28.342246</td>\n",
+       "      <td>-28.342246</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>110</th>\n",
+       "      <td>210</td>\n",
+       "      <td>Weekday</td>\n",
+       "      <td>168</td>\n",
+       "      <td>112</td>\n",
+       "      <td>-50.000000</td>\n",
+       "      <td>-50.000000</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>178</th>\n",
+       "      <td>28</td>\n",
+       "      <td>Weekday</td>\n",
+       "      <td>166</td>\n",
+       "      <td>107</td>\n",
+       "      <td>-55.140187</td>\n",
+       "      <td>-55.140187</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>8</th>\n",
+       "      <td>105</td>\n",
+       "      <td>Weekday</td>\n",
+       "      <td>193</td>\n",
+       "      <td>122</td>\n",
+       "      <td>-58.196721</td>\n",
+       "      <td>-58.196721</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>153</th>\n",
+       "      <td>251</td>\n",
+       "      <td>Weekday</td>\n",
+       "      <td>167</td>\n",
+       "      <td>101</td>\n",
+       "      <td>-65.346535</td>\n",
```

Comment:
You can see here the same mistake that I made in my initial pass at this: a bunch of these have what appear to be negative losses (i.e., service increases). This is due to Metro combining 105/705, 251/751, etc. So to account for that we need to combine those lines in the before/after picture.